xcell/mappers/mapper_DESY1wl.py

```diff
@@ -67,23 +75,24 @@ def _load_catalog(self):
             cat_zbin = Table.read(self.config['zbin_cat'],
                                   format='fits', memmap=True)
             cat_zbin.keep_columns(columns_zbin)
-            self.cat_data = astropy.table.join(self.cat_data, cat_zbin)
-            # Note: By default join uses checks the values of the column named
-            # the same in both tables
-            col_w = Column(name='weight',
-                           data=np.ones(len(self.cat_data), dtype=int))
-            self.cat_data.add_column(col_w)
+            self.cat_data = np.hstack((self.cat_data, cat_zbin()
 
             # remove bins which are not the one of interest
-            self.cat_data.remove_rows(self.cat_data['zbin_mcal'] != self.zbin)
+            # Logic: If item in one of zbins --> sel = False
+            # Thus it is not removed by next line
+            sel = self.cat_data['zbin_mcal'] != self.zbin and \ 
```

Comment:
Does this work? Doesn't it complain with `ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`? If so, change `and` to `*` (i.e. multiply the arrays).